index.js

```diff
@@ -226,12 +235,11 @@ app.get('/api/v1/convert', function(req,res) {
 function finishConversion(res,result,payload){
 	res.set('Access-Control-Allow-Origin','*');
 	res.set('Content-Type',payload.contentType);
-	if (payload.yaml) {
+	if (payload.responseYaml) {
```

Comment:
I think this should be `payload.yamlResponse` ?

Reply:
Absolutely.